posthog-node/src/extensions/sentry-integration.ts

```diff
@@ -0,0 +1,123 @@
+/**
+ * @file Adapted from [posthog-js](https://github.com/PostHog/posthog-js/blob/8157df935a4d0e71d2fefef7127aa85ee51c82d1/src/extensions/sentry-integration.ts) with modifications for the Node SDK.
+ */
+import { type PostHog } from '../posthog-node'
```

Comment:
Writing a generic integration would have worked by importing `PostHogCore`, but because the APIs differ quite a bit between web and Node, I decided to make this simpler for now by just implementing the Node version.